src/transforms/metric_to_log.rs

```diff
@@ -18,14 +18,15 @@ use crate::{
 /// Configuration for the `metric_to_log` transform.
 #[configurable_component(transform("metric_to_log"))]
 #[derive(Clone, Debug, Default)]
-#[serde(default, deny_unknown_fields)]
```

Comment:
Telling `serde` to use a default value for deserializing the configuration of a typed component makes no sense, because these types are never a fixed field... only part of a field using their containing enums i.e. `Transforms`.

It does, however, lead to the `vector-config-macros` code thinking that there's a default value that should be added to the configuration schema, which again... does not make sense at _this_ particular level.